fixity/storage_service.py

```diff
@@ -23,7 +24,7 @@ class StorageServiceError(Exception):
     itself will not return, but the caller still needs access to it.
     """
 
-    def __init__(self, message, report=None):
+    def __init__(self, message: str, report: Dict[str, str] = None) -> None:
```

Comment:
Why is this typing necessary if you're only extending test coverage?

Reply:
Well, it was necessary with the old assertion. So I am going to remove it in the next commit. Thanks for the suggestion!